src/systems/filecoin_mining/storage_mining/storage_miner_actor.go

```diff
@@ -589,14 +589,16 @@ func (a *StorageMinerActorCode_I) _isSealVerificationCorrect(rt Runtime, onChain
 
 	// TODO: serialize method param as {sectorSize,  DealIDs...}.
 
-	receipt := rt.SendCatchingErrors(&msg.InvocInput_I{
+	receipt, exitCode := rt.SendCatchingErrors(&msg.InvocInput_I{
 		To_:     addr.StorageMarketActorAddr,
-		Method_: storage_market.MethodGetUnsealedCIDForDealIDs,
+		Method_: storage_market.Method_StorageMarketActor_GetUnsealedCIDForDealIDs,
 		Params_: params,
 	})
 
-	if receipt.ExitCode() == exitcode.InvalidSectorPacking {
+	if exitCode.Equals(exitcode.UserDefinedError(exitcode.InvalidSectorPacking)) {
 		return false
+	} else if exitCode.IsError() {
+		panic("TODO")
```

Comment:
@zixuanzh I left this as a TODO; need to resolve correct behavior here

Reply:
looks like this method has evolved into `GetPieceInfosForDealIDs` and it does not return an exit code of InvalidSectorPacking. I think we should just use `SendPropagatingErrors` here and get back PieceInfos.

Reply:
OK -- in that case, probably best to defer until we can resolve the merge conflicts.